src/nvim/api/vim.c

```diff
@@ -2615,3 +2616,29 @@ void nvim__put_attr(Integer id, Integer start_row, Integer start_col,
   decorations_add_luahl_attr(attr, (int)start_row, (colnr_T)start_col,
                              (int)end_row, (colnr_T)end_col);
 }
+
+/// Gets api function documentation
+//
+// @return Dictionary of the form
+// { "nvim_win_set_option": {...}, "nvim_win_get_option": {...}, ... }
+Dictionary nvim_api_get_doc(void)
+  FUNC_API_SINCE(7)
+{
+    msgpack_unpacked unpacked;
+    msgpack_unpacked_init(&unpacked);
+    if (msgpack_unpack_next(&unpacked,
+                            (const char *)api_doc_metadata,
+                            sizeof(api_doc_metadata),
+                            NULL) != MSGPACK_UNPACK_SUCCESS) {
+        abort();
+    }
+
+    Object api_doc_obj;
+    msgpack_rpc_to_object(&unpacked.data, &api_doc_obj);
+    msgpack_unpacked_destroy(&unpacked);
+
+    Dictionary api_doc = ARRAY_DICT_INIT;
+    PUT(api_doc, "api_functions", DICTIONARY_OBJ(api_doc));
```

Comment:
`"api_functions"` is good or should that be changed?

Reply:
maybe just "api" works